wunderio/drupal-project
# WD-254: Upgrade Drupal core to 10.2.5, update other Composer packages

Ticket: WD-254

## Changes

- WD-254: Upgrade Drupal core to 10.2.5, update other Composer packages.
- WD-254: Update `basic` profile.

## Todo

I see no value in a custom `basic` profile.  We should move to `minimal` profile and move the configuration to `config/sync` instead. We have the configuration sync directory path defined in `settings.php` and the status page shows the error mesage _The `../config/sync` folder does not exist._

Maintaining custom profile configuration changes requires more work than regular configuration updates via `drush cex`.

I was using the following commands to clean up the profile config from `uuid:` & `_core:` keys: 

```sh
find web/profiles/custom/basic/config/install/ -type f -exec sed -i '' '/^uuid: /d' {} \;
find web/profiles/custom/basic/config/install/ -type f -exec sed -i '' '/_core:/{N;d;}' {} \;
```

## Testing

1. Install the site by running the following command: `lando drush si basic`.
2. Run `lando drush deploy` and make sure that Composer packages are upgraded.
